opencl/CMakeLists.txt

```
@@ -78,4 +78,12 @@ endif()
 FetchContent_MakeAvailable(ocl-icd)
 add_library(OpenCL-ICD ALIAS OpenCL)
 
+# Deploy OpenCL loader into package, this is required for OCL CPU test.
+if (NOT INTEL_OPENCL_STANDALONE_PRODUCT)
```

```suggestion
if (INTEL_OPENCL_STANDALONE_PRODUCT)
```
I think we should install ICD-Loader only if we build OpenCL (as a standalone product?). Otherwise, we built DPC++ and DPC++ is not supposed to deploy a copy of OpenCL-ICD loader. Did I get the meaning of `INTEL_OPENCL_STANDALONE_PRODUCT` variable correctly?

NOTE: Level Zero loader is installed as a separate package i.e. it's not installed together with Level Zero implementation. I think this approach has benefits when you install multiple implementations. If I install two OpenCL implementations (e.g. NVIDIA GPU and Intel CPU) each of them installs its own copy of ICD loader, which usually conflict with each other and depending on installation order one of implementation might be not available. 
I think we should consider deploying OpenCL-ICD loader separately from OpenCL CPU implementation in the future.

---

Currently if we do not install ICD-Loader, I think it will break a lot of tests on CPU backend.  The installation package / pseudo depends on the ICD loader we installed for testing.

---

> Currently if we do not install ICD-Loader, I think it will break a lot of tests on CPU backend. The installation package / pseudo depends on the ICD loader we installed for testing.

Okay. Let's deal with this later. What about the question about `INTEL_OPENCL_STANDALONE_PRODUCT` variable use?

---

If we remove "NOT" from line 82,  It will not install OpenCL library (libOpenCL.so) to lib folder in SYCLOS.  We have two concerns:  one is user needs to download and install OpenCL library by themselves.  Another concern is we may introduce the new features or changes in the OpenCL ICD we used to users.  If we do not install OpenCL library. We will miss some new features from OpenCL library.